src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -565,7 +553,32 @@ public static String calculatePathMD5(final Path path) throws IOException{
         if (!Files.isRegularFile(path)) {
             throw new IOException("File '" + fname + "' exists but is not a regular file");
         }
-        return Utils.calcMD5(Files.readAllBytes(path));
+        try {
+            return Utils.calcMD5(Files.readAllBytes(path));
+        } catch (final OutOfMemoryError e) {
```

Comment:
This is dangerous.  It's a clever idea, but OutOfMemory error isn't predictable.  It's very possible that you start doing this operation in this thread and somewhere else we get an OutOfMemoryError.  In general it's good advice to never catch `Error`.  They indicate problems which you are not expected to be able to recover from.

Reply:
I somehow missed this in my last review.  Sorry about that!

Reply:
Changed to always use a buffer when hashing files.